src/pages/learn/introspection.mdx

```diff
@@ -1,23 +1,41 @@
 # Introspection
 
-It's often useful to ask a GraphQL schema for information about what
-queries it supports. GraphQL allows us to do so using the introspection
-system!
+<p className="learn-subtitle">Learn how to query information about a GraphQL schema</p>
 
-For our Star Wars example, the file
-[starWarsIntrospection-test.ts](https://github.com/graphql/graphql-js/blob/main/src/__tests__/starWarsIntrospection-test.ts)
-contains a number of queries demonstrating the introspection system, and is a
-test file that can be run to exercise the reference implementation's
-introspection system.
+It's often useful to ask a GraphQL schema for information about what operations it supports. GraphQL allows us to do so using the [introspection system](https://spec.graphql.org/draft/#sec-Introspection).
 
-We designed the type system, so we know what types are available, but if
-we didn't, we can ask GraphQL, by querying the `__schema` field, always
-available on the root type of a Query. Let's do so now, and ask what types
-are available.
+Introspection queries are special kinds of queries that allow you to learn about a GraphQL API's schema, and they also help power GraphQL development tools. On this page, we'll learn how to run different queries to learn more about an underlying schema's types, fields, and descriptions.
+
+## Type name introspection
+
+We have already seen an example of introspection on the [Schemas and Types page](/learn/schema/). When querying a field that returned Union type, we included the `__typename` meta-field directly in a selection set to get the string value of the names of the different types returned by a search query. Let's look at this example again:
+
+```graphql
+# { "graphiql": true }
+query {
+  search(text: "an") {
+    __typename
+    ... on Character {
+      name
+    }
+    ... on Starship {
+      name
+    }
+  }
+}
+```
+
+We didn't add the `__typename` field to our GraphQL API explicitly—the GraphQL specification says that it must be provided to clients by a GraphQL implementation. This field can be queried for any field with an Object, Interface, or Union type as an output type.
```

Comment:
This text suffers the same issue that we hit quite a lot across the ecosystem, which is that really we're referring to the "unwrapped type" being Object/Interface/Union since it's also valid on types like `[[[User!]!]!]!`. If you can come up with a natural way of solving this without being overly technical, I'd love to hear it!

Reply:
We could probably just provide another example for illustrative purposes instead of getting too into the weeds with a written explanation. Maybe something like this, then call out that it can also provide the underlying type names when dealing with wrapped types:

```graphql
query {
  hero {
    name
    friends {
      __typename
      name
    }
  }
}
```

Reply:
Another example would certainly clarify, but perhaps we can just insert some subtle qualifier that makes it more precise without needing additional content - clearly "unwrapped output type" is too technical but maybe we can use something like "the ultimate output type" or "the underlying output type" or similar?

```suggestion
We didn't add the `__typename` field to our GraphQL API explicitly—the GraphQL specification says that it must be provided to clients by a GraphQL implementation. This field can be queried for any field with an Object, Interface, or Union type as the underlying output type.
```